manual/update-jenkins.md

```diff
@@ -0,0 +1,104 @@
+# Update Jenkins
+
+Jenkins needs to be kept up to date in several ways: we need to update the EC2
+instance it runs on, the Jenkins application itself, and all of its plugins.
+
+All the code needed to update Jenkins is in the [tdr-jenkins] repo.
+
+## Preparation
+
+**Before running any updates**:
+
+* Run the [TDR Jenkins Backup]
```

Comment:
This link goes to the Jenkins git repo. Is that correct, as there doesn't appear to be any instructions here on running the backup.

Reply:
Citrix copy-paste fail. Fixed, thanks.